core/src/html/layout.rs

```diff
@@ -660,10 +660,10 @@ impl<'a, 'gc> LayoutContext<'a, 'gc> {
             {
                 return font;
             }
-            // TODO: If set to use embedded fonts and we couldn't find any matching font, show nothing
-            // However - at time of writing, we don't support DefineFont4. If we matched this behaviour,
-            // then a bunch of SWFs would just show no text suddenly.
```

Comment:
just to be sure, did you test some SWFs like this?

Reply:
Generally yes, I made sure that Flash behaves this way and most SWFs from my text collection work.

However, I've just remembered one SWF that used a wrong font: https://github.com/ruffle-rs/ruffle/issues/14740, and unfortunately checking it now, it turns out the text disappears in it. Which makes sense, because now Ruffle somehow fails to load the embedded font and falls back to a device font \:/

So I guess there are (and probably will be) cases when Ruffle fails to load an embedded font. Some probably are hard to spot.